apps/synt-sykemelding-api/src/main/resources/application.yml

```diff
@@ -9,7 +9,9 @@ consumers:
   hodejegeren:
     url: http://testnorge-hodejegeren.default.svc.nais.local
   helsepersonell:
-    url: http://testnorge-helsepersonell-api.default.svc.nais.local
+    client_id: af686685-d25a-462a-a0ea-e2a348c457d3
+    client_secret: ${HELSEPERSONELL_API_CLIENT_SECRET}
```

Comment:
Var det denne Kristen sa ble endret til `${azure.app.client.secret} ` for å hente etter deploy?